BUILDING.md

```diff
@@ -106,16 +106,19 @@ On macOS, you will need to install the `Xcode Command Line Tools` by running
 installed, you can find them under the menu `Xcode -> Open Developer Tool ->
 More Developer Tools...`. This step will install `clang`, `clang++`, and
 `make`.
-* After building, you may want to setup [firewall rules](tools/macosx-firewall.sh)
-to avoid popups asking to accept incoming network connections when running tests:
 
 If the path to your build directory contains a space, the build will likely fail.
 
+After building, setting up [filewall rules](tools/macosx-firewall.sh) can avoid popups
```

Comment:
I'm assuming this was a typo in the suggestion from @jasnell and that this should have remained as `firewall rules`.